modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -1361,6 +1365,170 @@
         }
       }
     },
+    "/:veteranId/526": {
+      "post": {
+        "summary": "Submits form 526",
+        "tags": [
+          "Disability"
+        ],
+        "operationId": "post526Claim",
+        "security": [
+          {
+            "productionOauth": [
+              "claim.read"
+            ]
+          },
+          {
+            "sandboxOauth": [
+              "claim.read"
```

Comment:
submits a 526 with claim.read? this seems backwards. shouldn't a submit endpoint be claim.write? 

we'll also be moving to `system/claim.read` and `system/claim.write` when Austin's changes go in. This will probably be updated once we do that - fine with staying `claim.read` and `claim.write` for now though.

Reply:
Good catch!